scripts/ssh-hostkey.nse

```diff
@@ -316,24 +345,30 @@ local function portaction(host, port)
       bits=key.bits,
       key=key.key,
     }
-    if format:find( 'hex', 1, true ) or all_formats then
-      table.insert( output, ssh1.fingerprint_hex( key.fingerprint, key.algorithm, key.bits ) )
-    end
-    if format:find( 'bubble', 1, true ) or all_formats then
-      table.insert( output, ssh1.fingerprint_bubblebabble( openssl.sha1(key.fp_input), key.algorithm, key.bits ) )
+    -- Check for bad keys	
+    local bad_key=bin.pack("H",out.fingerprint)
+    if result[bad_key]~= out.fingerprint then
```

Comment:
We still want to produce the old output even if the key is found in the "bad" list.